cmd/edit_dt.go

```diff
@@ -0,0 +1,212 @@
+package cmd
```

Comment:
Lets not do edit for this release. I need to think more about this.

Reply:
okay for this release our main focus is dt assemble and not dt edit, right ?